web/client/translations/data.en-US

```diff
@@ -311,6 +311,16 @@
                 "title": "Maps ({count})",
                 "noMapAvailable": "No map available",
                 "createNewOne": "Create a new one"
+            },
+            "geostories": {
+                "newGeostory": "New Story",
+                "title": "Stories ({count})",
+                "titleNoCount": "Stories",
+                "create": "Create Story",
+                "noGeostoryAvailable": "No story available",
+                "createANewOne": "Create a new one",
+                "deleteError": "There was an error deleting the resource",
+                "errorLoadingDashboards": "There was an error loading stories"
```

Comment:
- rename  this 
- check if all this localized strings are used, otherwise remove the unused ones.
- the same for all languages